pages/challenge.js

```diff
@@ -12,6 +12,32 @@ const RepoLink = 'https://github.com/OperationCode/front-end/';
 const ChallengeLink = `${RepoLink}blob/main/pages/challenge.js`;
 const CompareLink = `${RepoLink}compare`;
 
+export const MakeColumnsFromArray = ({ ary }) => {
+  const numberNamesPerColumn = 20;
+  const numberCols = Math.floor(ary.length / numberNamesPerColumn) + 1;
+
+  const content = [];
+  let col = 0;
+
+  while (col < numberCols) {
+    const colPersonStartIndex = col * numberNamesPerColumn;
+    const colPersonEndIndex = col * numberNamesPerColumn + numberNamesPerColumn;
+    const elements = challengers.slice(colPersonStartIndex, colPersonEndIndex);
+
+    content.push(
+      <div key={col} className={styles.challengerListColumn}>
+        {elements.map(element => (
+          <li key={element}>{element}</li>
+        ))}
+      </div>,
+    );
+
+    col += 1;
+  }
+
+  return content;
```

Comment:
You don't need to rewrite this. Mainly, I just enjoy showing people how to immutably construct mungings (no using `let` / `[].push()`). I would be happy if you took some of my variable renaming ideas though!

```jsx
  import range from 'lodash/range';
  
  //
  
  const numberOfNamesPerColumn = 20;
  const numberOfColumns = Math.floor(names.length / numberOfNamesPerColumn) + 1;
  const columns = range(1, numberOfColumns);

  const content = columns.map((columnNumber, index) => {
    const startIndex = index * numberOfNamesPerColumn;
    const endIndex = columnNumber * numberOfNamesPerColumn;
    const namesInColumn = challengers.slice(startIndex, endIndex);
    
    return (
      <ol key={columnNumber} className={styles.challengerListColumn}>
        {namesInColumn.map((name) => (
          <li key={name}>{name}</li>
        ))}
      </ol>,
    );
  });

  return content;
```